packages/lit-dev-content/samples/examples/react-basics/demo-greeting.ts

```diff
@@ -1,5 +1,5 @@
-import { html, LitElement } from 'lit';
-import { customElement, property } from 'lit/decorators.js';
+import {html, LitElement} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
 
 @customElement('demo-greeting')
```

Comment:
Maybe a comment like `// A plain Lit component, which can be used in any framework`?